src/utils/supabase/products.ts

```
@@ -0,0 +1,162 @@
+  
+  // Sort products
+  return [...filtered].sort((a, b) => {
+    switch (sortBy) {
+      case 'price_asc':
+        return a.price - b.price;
+      case 'price_desc':
+        return b.price - a.price;
+      case 'newest':
+      default:
+        // As with the hooks file, sort by product_id
+        const idA = parseInt(a.product_id);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)